### PR TITLE
Amls 5842 - Accessibility statement + footer link + toggle

### PR DESCRIPTION
**Pull request by matthewharris1**
There is 2 more PRs that are linked to this ticket. both linked below. One in EAB and one in AMP.
https://github.com/hmrc/amls-estate-agency-business-frontend/pull/41
https://github.com/hmrc/amls-art-market-participant-frontend/pull/74

## How Has This Been Tested?
Unit
Manual

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -200,6 +201,10 @@ microservice {
         with-js = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/problem_reports_ajax?service=AMLS"
         non-js = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/problem_reports_nonjs?service=AMLS"
       }
+      accessibility-statement-url {
+        authenticated = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/accessibility"
+        unauthenticated = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/accessibility"
```
Comment (ajchilts) [inline conf/application.conf:206]: We will need to look at adding these to app-config-base also. Need a PR for this.

File: app/controllers/accessibility/AccessibilityStatementController.scala [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import config.ApplicationConfig
+import controllers.{AmlsBaseController, CommonPlayDependencies}
+import javax.inject.Inject
+import play.api.mvc.{Call, MessagesControllerComponents}
+import utils.AuthAction
+import views.html.accessibility._
+
+import scala.concurrent.Future
+
+class AccessibilityStatementController @Inject()(authAction: AuthAction,
```
Comment (ajchilts) [inline app/controllers/accessibility/AccessibilityStatementController.scala:32]: We need to add a spec (test) for this controller

**Comment by ajchilts** [comment]
@Bongowhack  @mostafasina - We will need to test this in staging once merged and @matthewharris1 has created the config in app-config-base. 

We should also use staging to test that the toggle works. 

@matthewharris1 - I can help with the config if you get stuck.

File: conf/application.conf [hunk 3]
```diff
@@ -168,6 +168,7 @@ microservice {
       enrolment-store = true
       fx-enabled = true
       public-message-banner = true
+      accessibility-statement = true
```
Comment (ajchilts) [inline conf/application.conf:171]: We will need to add this to app-config-base

File: app/controllers/accessibility/AccessibilityStatementController.scala [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                                                 val ds: CommonPlayDependencies,
+                                                 val cc: MessagesControllerComponents,
+                                                 config: ApplicationConfig) extends AmlsBaseController(ds, cc) {
+
+  def get = authAction.async {
```
[the remark this answers is not in the document]
Comment (matthewharris1) [reply app/controllers/accessibility/AccessibilityStatementController.scala]: Done

File: app/controllers/accessibility/AccessibilityStatementController.scala (hunk 2 above)
Comment (matthewharris1) [reply app/controllers/accessibility/AccessibilityStatementController.scala]: Done

File: conf/application.conf (hunk 1 above)
Comment (matthewharris1) [reply conf/application.conf:206]: Done

File: app/views/accessibility/accessibility_statement.scala.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+    <h2>@Messages("accessibility.statement.heading.seven")</h2>
+    <p>@Messages("accessibility.statement.paragraph.seventeen")</p>
+    <p>@Messages("accessibility.statement.paragraph.eighteen") <a href="https://www.w3.org/TR/WCAG21/">@Messages("accessibility.statement.paragraph.eighteen.link")</a> </p>
+
+    <h2>@Messages("accessibility.statement.heading.eight")</h2>
+    <p>@Messages("accessibility.statement.paragraph.nineteen", s"${appConfig.accessibilityStatementUpdated}")</p>
+    <p>@Messages("accessibility.statement.paragraph.twenty")</p>
+    <p>@Messages("accessibility.statement.paragraph.twentyone", s"${appConfig.accessibilityStatementTested}"))</p>
+
```
Comment (slavvco) [inline app/views/accessibility/accessibility_statement.scala.html:80]: duplicated )) bracket

File: conf/messages [hunk 6]
```diff
@@ -2782,3 +2782,59 @@ login-event.skip-for-now = Skip for now and continue to the service
...
+accessibility.statement.paragraph.thirteen.link2 = Equality Commission for Northern Ireland (ECNI)
+accessibility.statement.paragraph.fourteen = We provide a text relay service if you are deaf, hearing impaired or have a speech impediment.
+accessibility.statement.paragraph.fifteen = We can provide a British Sign Language (BSL) interpreter, or you can arrange a visit from an HMRC advisor to help you complete the service.
+accessibility.statement.paragraph.sixteen = Find out how to
+accessibility.statement.paragraph.sixteen.link = contact us
+accessibility.statement.paragraph.seventeen = HMRC is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+accessibility.statement.paragraph.eighteen = This service is fully compliant with the
+accessibility.statement.paragraph.eighteen.link = Web Content Accessibility Guidelines version 2.1 AA standard
+accessibility.statement.paragraph.nineteen = The service was last tested on Monday {0} and was checked for compliance with WCAG 2.1 AA.
```
Comment (slavvco) [inline conf/messages:2818]: ... last tested on Monday... what if the date will be Tuesday?

File: conf/application.conf [hunk 7]
```diff
@@ -288,3 +290,8 @@ auditing {
     }
   }
 }
+
+accessibility-statement {
+    updated = "11 May 2020"
+    tested = "Monday 15 May 2020"
```
Comment (slavvco) [inline conf/application.conf:296]: "Monday" is not needed in conf?

Comment (matthewharris1) [reply conf/application.conf]: Removed

File: conf/messages (hunk 6 above)
Comment (matthewharris1) [reply conf/messages]: Changed

File: app/config/ApplicationConfig.scala [hunk 8]
```diff
@@ -123,4 +128,8 @@ class ApplicationConfig @Inject()(configuration: Configuration, runMode: RunMode
   lazy val payBaseUrl = s"${baseUrl("pay-api")}/pay-api"
 
   lazy val businessMatchingUrl = s"${baseUrl("business-customer")}/business-customer"
+
+
+  def accessibilityStatementUpdated = servicesConfig.getConfString("accessibility-statement.updated", "11th May 2020")
+  def accessibilityStatementTested = servicesConfig.getConfString("accessibility-statement.tested", "15th May 2020")
```
[the remark this answers is not in the document]
Comment (matthewharris1) [reply app/config/ApplicationConfig.scala:133]: These were the dates provided by Martin/Lisa, can be changed whenever needed

File: conf/application.conf (hunk 3 above)
Comment (matthewharris1) [reply conf/application.conf:171]: added

**Pull request merged**